source/common/dynamo/dynamo_filter.h

```diff
@@ -27,7 +27,8 @@ class DynamoFilter : public Http::StreamFilter {
   }
 
   // Http::StreamFilterBase
-  void onDestroy() override {}
+  void setBufferLimit(uint32_t limit) override { buffer_limit_ = limit; }
+  void onDestroy() override { stream_reset_ = true; }
```

Comment:
I think this is required for sendLocalReply on the event the stream is destroyed mid-call.
see source/common/http/utility.h and https://github.com/lyft/envoy/issues/1283